augur/dates/__init__.py

```diff
@@ -5,9 +5,10 @@
 import pandas as pd
 import re
 import treetime.utils
-from .errors import AugurError, InvalidDate
+from augur.errors import AugurError
+from .errors import InvalidDate
 
-from augur.util_support.date_disambiguator import DateDisambiguator
+from .date_disambiguator import DateDisambiguator
```

Comment:
_non-blocking_

Since this module is now in the `augur.dates` package, I'd drop the `date_` prefix in the module name to make it `augur.dates.disambiguator`. Ok to leave the class inside as `DateDisambiguator`.

Reply:
Hmm, in that case I'd be inclined to cherry-pick ef9c3f69c9a22f91b180e5fbc3a0e464355bd6d6 into this PR. What do you think of that rename?

Reply:
Sure, that's fine. Would be nice to have it only move once not twice. :-) That branch is already based on this one, yeah?

Reply:
Ok, added to this PR. And yeah, it is.

Reply:
Oops, I merged ef9c3f69c9a22f91b180e5fbc3a0e464355bd6d6 as-is but forgot that there was [another discussion](https://github.com/nextstrain/augur/pull/1144#discussion_r1092575174) about the commit description...

In case anyone is confused about the wording and somehow finds this comment, what I meant was:

> "DateDisambiguator" is a mouthful, and uses a verb-based noun in a class name which is a bad pattern (might as well be a single function named disambiguate_date). Use a more "definite" noun instead.

Reply:
Ahh, no worries at all!